framework/src/modules/chain/index.js

```diff
@@ -67,6 +67,10 @@ class ChainModule extends BaseModule {
 			'delegates:fork',
 			'loader:sync',
 			'dapps:change',
+			'process:newBlock',
```

Comment:
Maybe `process` is too generic. We could propose a list of names for this.
I propose:
- `blocks_processor`
- `processor` <- although it continues being still generic

Reply:
yea probably it should be `processor`